cheatSheets.md

```diff
@@ -48,3 +48,4 @@
 
 - [HTML Cheat Sheet](https://html.com/wp-content/uploads/html-cheat-sheet.pdf): This clear, easy-to-access PDF thoroughly explains HTML elements, with understandable explanations and examples.
 
+- [Arduion Programming Cheat Sheet](https://github.s3.amazonaws.com/downloads/liffiton/Arduino-Cheat-Sheet/Arduino%20Cheat%20Sheet.pdf):Arduion Programming Cheat Sheet covering all important syntax.
```

Comment:
@Subathra19 I think there is a little typo. Is `Arduion` meant to say Arduino? Thank you.

Reply:
@CodingSpecies Good catch.

Reply:
@CodingSpecies  Thanks for the comment. Yes, its a typo